notebooks/Notebook2.md

```diff
@@ -2,33 +2,56 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+“I think it would be useful to ensure you have some documentation of this structure both for: 1) your own reference, which may allow you to reflect on what other use cases you may want in your design and 2) users writing in your DSL for the first time, it will allow them to have a template (and some example calls would be good) to look at when they are learning to write out commands in your DSL.”
+
+“Let me know if you want to talk about the repo at all, or if there is something specific I can look at.”-Prof Ben
+
+“I liked that you noted that the work was tedious despite not being complex, is there any way that you think you could have implemented these that would have been less tedious?”
+
+I will address the three critiques above:
+
+I agree. In the end I think that I will need some form of documentation for the language that shows different possibilities for each input group of the language as well as the general structure. I think I will plan to work on that near the end of the project term. Right now I want to focus on other aspects. 
+
+To Prof Ben, I do think I may need some help with my repository. Although It has been working pretty well I ran into a problem this week when trying to implement a small interface for inputting my language. It turns out that codespaces does not know how to build a physical java application type interface without the proper setup in the repository or connections to a remote explorer. I would appreciate help with that sometime next week. 
+
```

Comment:
Yeah, if you're trying to run a Java app in codespaces, it can be tricky. I see that you have a `.devcontainer.json` file that should be set up for Java. We can try to debug it together in class on Wednesday, or during office hours!